whatWeDo/sectionHowItWorks.html

```diff
@@ -4,28 +4,28 @@
     <div class="col-lg-10 col-lg-offset-1">
       <h2 class="h1">How it Works</h2>
     </div>
-    <div class="col-lg-2 col-lg-offset-1">
+    <div class="col-lg-3 col-lg-offset-1">
       <div class="row tab-nav">
         <ol class="tabs">
-          <li class="tab is-active">
-            <a href="#" class="" data-target="identifying">1. Identifying the Problems</a>
+          <li class="tab">
+            <a href="#" class="button" data-target="identifying">1. Identifying the Problems</a>
```

Comment:
There are already .btn classes defined in landing/client/style, they should be reused